Dockerfile installs wrong version of mmcv

**Describe the bug**
Installing MMOCR via the Dockerfile provided installs the wrong version of MMCV, causing the following error when running the Demo tool:

```
Traceback (most recent call last):
  File "mmocr/utils/ocr.py", line 16, in <module>
    from mmocr.apis import init_detector
  File "/mmocr/mmocr/__init__.py", line 5, in <module>
    import mmdet
  File "/opt/conda/lib/python3.7/site-packages/mmdet/__init__.py", line 25, in <module>
    f'MMCV=={mmcv.__version__} is used but incompatible. ' \
AssertionError: MMCV==1.4.4 is used but incompatible. Please install mmcv>=1.3.8, <=1.4.0.
```

**Reproduction**

1. What command or script did you run?

After building the docker image with the current Dockerfile, inside the container run:

```bash
python mmocr/utils/ocr.py data/mydata --det None --recog CRNN_TPS --batch-mode --single-batch-size 10 --output data/mydata_out
```

2. Did you make any modifications on the code or config? No
3. What dataset did you use? Irrelevant

**Environment**

1. Please run `python mmocr/utils/collect_env.py` to collect necessary environment information and paste it here.

Can't do:

```
Traceback (most recent call last):
  File "mmocr/utils/collect_env.py", line 5, in <module>
    import mmocr
  File "/mmocr/mmocr/__init__.py", line 5, in <module>
    import mmdet
  File "/opt/conda/lib/python3.7/site-packages/mmdet/__init__.py", line 25, in <module>
    f'MMCV=={mmcv.__version__} is used but incompatible. ' \
AssertionError: MMCV==1.4.4 is used but incompatible. Please install mmcv>=1.3.8, <=1.4.0.
```

2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]: Dockerfile

**Error traceback**
```
Traceback (most recent call last):
  File "mmocr/utils/collect_env.py", line 5, in <module>
    import mmocr
  File "/mmocr/mmocr/__init__.py", line 5, in <module>
    import mmdet
  File "/opt/conda/lib/python3.7/site-packages/mmdet/__init__.py", line 25, in <module>
    f'MMCV=={mmcv.__version__} is used but incompatible. ' \
AssertionError: MMCV==1.4.4 is used but incompatible. Please install mmcv>=1.3.8, <=1.4.0.
```

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -15,7 +15,7 @@ RUN apt-get update && apt-get install -y git ninja-build libglib2.0-0 libsm6 lib
 RUN conda clean --all
 RUN pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html
 
-RUN pip install mmdet==2.14.0
+RUN pip install mmdet==2.20.0
 
 RUN git clone https://github.com/open-mmlab/mmocr.git /mmocr
 WORKDIR /mmocr

